### ISSUE TITLE
The github action failed to post reports

### ISSUE THREAD

**Issue by joyeecheung** [nodejs/reliability#355]
From a recent run:

```
Run title_date=$(date +%Y-%m-%d)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 69480  100   296  100 69184    736   168k --:--:-- --:--:-- --:--:--  168k
{
  "message": "Validation Failed",
  "errors": [
    {
      "resource": "Issue",
      "code": "custom",
      "field": "body",
      "message": "body is too long (maximum is 65536 characters)"
    }
  ],
  "documentation_url": "https://docs.github.com/rest/reference/issues#create-an-issue"
}
100 69480  100   296  100 69184    736   168k --:--:-- --:--:-- --:--:--  168k
```

I am guessing that the recent debugger tests fallouts resulted in a super long report that couldn't be posted in one go due to rescritions of GitHub. We probably need to update it so that it split different parts into replies in that case.

**Comment by joyeecheung** [comment]
Another thing it could do is to commit that report into this repository, and post a link to that document.

**Comment by joyeecheung** [comment]
Looks like the report is short enough to be posted again: https://github.com/nodejs/reliability/issues/356

But it may be good to always create a file in this repo anyway, for posterity. I found the doc on how to create the file using the RESTful API:https://docs.github.com/en/rest/repos/contents#create-or-update-file-contents

**Comment by richardlau** [comment]
I have a PR that pastes the generated markdown into GitHub Actions step summary for the job: https://github.com/nodejs/reliability/pull/274
I don't know if the step summaries have similar size limits. I also suspect they won't persist as action logs eventually get purged.

**Comment by joyeecheung** [comment]
Indeed, it seems the summary from https://github.com/nodejs/reliability/actions/runs/2314866636 is already unavailable.

**Comment by joyeecheung** [comment]
I think for summaries we could also just post the stats (i.e. cut everything up to the `### Jenkins Failure` section).
